activiti-cloud-service-common-properties/src/main/java/org/activiti/cloud/common/properties/CommonPropertiesConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package org.activiti.cloud.common.properties;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:conf/common-configuration.properties")
+public class CommonPropertiesConfiguration {
```

Comment:
@miguelruizdev can you update the class suffix from `Configuration` to `AutoConfiguration`?

Reply:
@erdemedeiros is this a Configuration or an AutoConfiguration. In spring boot the distiction is to include the auto configurations into the meta-inf/spring.factories (if I remember correctly) file.

Reply:
It's an AutoConfiguration, it's referenced in the `spring.factories` just below.

Reply:
totally agree.